packages/settingeditor/src/pluginlist.tsx

```diff
@@ -451,7 +451,7 @@ export class PluginList extends ReactWidget {
         <FilterBox
           updateFilter={this.setFilter}
           useFuzzyFilter={false}
-          placeholder={trans.__('Search…')}
+          placeholder={trans.__('Search settings…')}
```

Comment:
are these ellipsis functional? do they get announced on assistive tech at all?

Reply:
Hey @tonyfast, they are not functional and don't get announced on assistive tech. I kept the ellipsis as it was used with the previous placeholder text. Would you like me to remove it? 

Looking at [this UX article](https://ux.stackexchange.com/questions/148512/should-i-include-at-the-end-of-input-placeholders), it might be best if I remove it.

Reply:
thanks for looking further into this. that article is a good reference. i don't feel strongly either way. i think that if the pattern is consistent with the code base, and it doesn't lead to screen reader chatter, then it is fine.

i do like the `...` differentiation meaning to expect a follow up action, but it seems nuanced if not consistently implemented.

Reply:
Thanks @tonyfast. It's consistent with the codebase so I'll keep it for now.